doc/source/amazon/step-zero-aws.rst

```diff
@@ -7,166 +7,162 @@ AWS does not have native support for Kubernetes, however there are
+         https://github.com/kubernetes/kops/blob/master/docs/run_in_existing_vpc.md
+       --master-count
+         Spawns more masters in one or more VPCs
+         This improves redudancy and reduces downtime during cluster upgrades
+       --master-zones
+         specify zones to run the master in
+       --node-count
+         Increases the total nodes created (default 2)
+       --master/node-security-groups
+         Allows you to specify additional security groups to put the masters and nodes in by default
+       --ssh-access
+         By default SSH access is open to the world (0.0.0.0).
+         If you are using a private topology, this is not a problem.
+         If you are using a public topology make sure your ssh keys are strong and you keep sshd up to date on your cluster's nodes.
+
+10. Wait for the cluster to start-up
+
+    Running the 'kops validate cluster' command will tell us what the current state of setup is.
+    If you see "can not get nodes" initially, just be patient as the cluster can't report until a
+    few basic services are up and running.
+   
+    Keep running 'kops validate cluster' until you see "Your cluster $NAME is ready" at the end of the output.
+   
+    ``time until kops validate cluster; do sleep 15 ; done`` can be used to automate the waiting process.
+    
+    If at any point you wish to destroy your cluster after this step, run ``kops delete cluster $NAME --yes``
+    
+
+11. Confirm that ``kubectl`` is connected to your Kubernetes cluster.
+
+    Run::
+
+       kubectl get nodes
+
+    you should see a list of two nodes, each beginning with ``ip``.
+
+12. Configure ssh bastion
+
+    Skip this step if you did not go with the private option above!
+   
+    Ideally we would simply be passing the --bastion flag into the kops command above.  However that flag is not functioning as intended at the moment.  https://github.com/kubernetes/kops/issues/2881
+   
+    Instead we need to follow this guide: https://github.com/kubernetes/kops/blob/master/docs/examples/kops-tests-private-net-bastion-host.md#adding-a-bastion-host-to-our-cluster
+    
+    At this point there are a few public endpoints left open which need to be addressed
+    
+    * Bastion ELB security group defaults to access from 0.0.0.0
+    * API ELB security group defaults to access from 0.0.0.0
+      
+
+13. Enable dynamic storage on your Kubernetes cluster.
+    Create a file, ``storageclass.yml`` on your local computer, and enter
+    this text::
+
+        kind: StorageClass
+        apiVersion: storage.k8s.io/v1
+        metadata:
+          annotations:
+             storageclass.beta.kubernetes.io/is-default-class: "true"
+          name: gp2
+        provisioner: kubernetes.io/aws-ebs
+        parameters:
+          type: gp2
+
+    Next, run this command:
+
+        .. code-block:: bash
+
+           kubectl apply -f storageclass.yml
+
+    This enables `dynamic provisioning
```

Comment:
In a discussion with @willingc, she thought that including the instructions for  creating an encrypted storage class would be useful in the documentation.
An easy way to do this is:
1) add to the storageclass.yml file in the parameters section:
 encrypted: "true"  .. should note that the quotes are important
2)  run kubectl delete storageclass gp2   --- This step is important as you can not modify a storage class apparently
3)  kubectl appy -f storageclass.yml

Reply:
Can we defer this to a future PR?  I'd like to keep this as close to the original as possible.

Reply:
Sure. Thought since I was in here to add it.